src/admin/views/activities/view.html.php

```diff
@@ -60,7 +69,14 @@ public function display($tpl = null)
 	 */
 	protected function addToolBar()
 	{
-		$title = JText::_('Activity Stream');
+		$title        = JText::_('Activity Stream');
+		$input        = JFactory::getApplication()->input;
+		$languageFile = JFactory::getLanguage();
+		$extension    = JFactory::getApplication()->input->get('client', '', 'word');
```

Comment:
@Mayaank12 check the datatype (third param)

Reply:
done